### PR TITLE
Add Hai Reverse Thrust Oufits

### PR DESCRIPTION
**Pull request by Hecter94**
**Content (Artwork / Missions / Jobs)**

## Summary
Adds a tiny, small and medium reverse thruster for the Hai. Using the same stats as the regular thrusters, but reverse.
Additionally adds a reverse thruster to the Geocoris and Water Bug. 
I looked at variants as well, but they all looked to be more "militarized" versions of the base ship so I didn't feel adding reverse thrust was appropriate.

<details>
<summary>Chart showing how these compare to other sources of reverse thrust:</summary>

chart (1)

Full stats [here](https://docs.google.com/spreadsheets/d/1ilIAl_HxctUEHTvoQgiFw2vsG1kx2iwaad4ojZ1K9jA/edit?usp=sharing)
</details>

## Save File
This save file will place you on Hai-Home with several Hai ships and access to all three reverse thrusters:
[Hai Reverse.txt](https://github.com/endless-sky/endless-sky/files/9689488/Hai.Reverse.txt)

## Testing Done
Ensured each ship is still under their outfit/weapon capacity and was able to reverse thrust as expected. Ensured the outfits showed up in the outfitter and ensured that the outfits were using the correct reverse stats.

Big thanks to @EjoThims for his help with getting the lighting and shading looking correct on the reverse thrusters. They look much better with his help.

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/87
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/308

### CODE REVIEW

File: data/hai/hai outfits.txt [hunk 1]
```diff
@@ -549,6 +549,50 @@ outfit `"Bufaer" Atomic Thruster`
...
+	"mass" 18
+	"outfit space" -18
+	"weapon capacity" -18
+	"reverse thrust" 12.5
+	"reverse thrusting energy" 1.4
+	"reverse thrusting heat" 2.8
+	"reverse flare sprite" "effect/atomic flare/tiny"
+		"frame rate" 14
+	"reverse flare sound" "atomic tiny"
+	description "This tiny reverse thruster allows even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```
Comment (EjoThims) [inline data/hai/hai outfits.txt:565]:
```suggestion
	description "This tiny reverse thruster allows even the smallest of Hai ships to mount it for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

Comment (EjoThims) [inline data/hai/hai outfits.txt:565]:
```suggestion
	description "These tiny reverse thrusters allow even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

Comment (EjoThims) [inline data/hai/hai outfits.txt:565]:
```suggestion
	description "This tiny reverse thruster allows even the smallest of Hai ships additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

First two depending on which way you want to go on number agreement there (but plural would be most consistent with other entries), but honestly it's a bit awkward either way; third just skips the problem entirely and is far smoother.

You could also trim the phrase and still switch it to plural to be consistent with other entries, if you prefer that.

File: data/hai/hai outfits.txt [hunk 2]
```diff
@@ -549,6 +549,50 @@ outfit `"Bufaer" Atomic Thruster`
...
+	"mass" 28
+	"outfit space" -28
+	"weapon capacity" -28
+	"reverse thrust" 22.4
+	"reverse thrusting energy" 2.5
+	"reverse thrusting heat" 4.8
+	"reverse flare sprite" "effect/atomic flare/tiny"
+		"frame rate" 14
+	"flare sound" "atomic tiny"
+	description "Many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on many larger Hai vessels. Considering them to be well worth the steep cost."
```
Comment (EjoThims) [inline data/hai/hai outfits.txt:580]:
```suggestion
	description "Considering them to be well worth the steep cost, many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on larger Hai vessels."
```

That's just a big ol hanging fragment.

Repetitive "many" can go, too.

File: data/hai/hai outfits.txt [hunk 3]
```diff
@@ -549,6 +549,50 @@ outfit `"Bufaer" Atomic Thruster`
...
+	"mass" 44
+	"outfit space" -44
+	"weapon capacity" -44
+	"reverse thrust" 39.4
+	"reverse thrusting energy" 4
+	"reverse thrusting heat" 8.2
+	"reverse flare sprite" "effect/atomic flare/small"
+		"frame rate" 13
+	"reverse flare sound" "atomic small"
+	description "This is the largest reverse thruster the Hai offer, preferring to focus on additional steering when more maneuverability is needed. Despite that, these reverse thrusters boast an impressive amount of power for their size, allowing even large ships to quickly come to a stop when needed."
```
Comment (EjoThims) [inline data/hai/hai outfits.txt:595]:
```suggestion
	description "With pilots generally preferring to focus on steering when actual agility is needed, these are the largest reverse thrusters the Hai offer. They still boast an impressive amount of power for their size, which allows even the largest Hai ships to quickly come to a stop when needed."
```
Another bad fragment. Simplest fix  would be to just add the mention of pilots, but rearranging too gets much smoother flow.

Went ahead and made this one plural as well.

File: data/hai/hai outfits.txt (hunk 2 above)
Comment (tibetiroka) [inline data/hai/hai outfits.txt:580]:
```suggestion
	description "Many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on many larger Hai vessels, considering them to be well worth the steep cost."
```
Still doesn't feel 'smooth' to me, but better than a separate sentence

**Review by EjoThims — CHANGES_REQUESTED** [review]
I'm also not sure if these should be 100% weapon space like human reversers or not, but don't have any specific aversions to that or recommendations on change for it, either.

**Review by tibetiroka — CHANGES_REQUESTED** [review]

**Review by tibetiroka — APPROVED** [review]

**Review by EjoThims — APPROVED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Review by GefullteTaubenbrust2 — APPROVED** [review]
There isn't a ton to say about the artwork, it's just the flipped versions of existing outfits, which is of course exactly what these should be. Maybe the medium one is a little too bright in the shadows? There isn't anything hugely wrong though that jumps out at me so it's an approval.

**Comment by Hecter94** [comment]
Requesting a review from Derpy on this now that it has several approvals.

**Pull request merged**